src/utils.js

```diff
@@ -6,3 +6,16 @@ export const setFilterOnly = x =>
     F.when(_.isPlainObject, F.setOn('filterOnly', true)),
     _.cloneDeep(x)
   )
+
+// Wraps provided node in a new `and` group, using the first node/tree's schema and key
+export let andGroup = (...args) => {
+  let children = _.flatten(args)
+  let [{ schema, key }] = children
+  return { key: `${key}-parent`, join: 'and', schema, children }
```

Comment:
I know `type: 'group'` isn't necessary but since it's usually there I think maybe the util should go for it

Reply:
done :)